### PR TITLE
chore: add updates to force redeployment on Vercel

### PR DESCRIPTION
**Pull request by Parkreiner**
## Changes made
- Updated `check.sh` script to add support for automatic re-deploying in the event that the the registry has a partial/full outage.

### CODE REVIEW

File: .github/scripts/check.sh [hunk 1]
```diff
@@ -17,6 +19,11 @@ for var in "${required_vars[@]}"; do
     fi
 done
 
+JUST_REDEPLOYED="$JUST_REDEPLOYED:-0"
+if (JUST_REDEPLOYED); do
+    return 1
```
Comment (bcpeinhardt) [inline .github/scripts/check.sh:24]: Maybe an echo for context here

File: .github/scripts/check.sh [hunk 2]
```diff
@@ -108,6 +152,20 @@ else
...
+    # ASAP
+    status_code=$(force_redeploy_registry)
+    # shellcheck disable=SC2181
+    if (( status_code == 200 )); then
+        echo "Reployment successful"
+    else
+        echo "Unable to redeploy automatically"
+    fi
+
+    # Update environment variable so that if automatic re-deployment fails, we
```
Comment (bcpeinhardt) [inline .github/scripts/check.sh:166]: Update this comment to specify a successful redeploy != things are all fixed.

**Review by johnstcn — APPROVED** [review]
made some changes and shellcheck no longer complains, so it "should" be fine 🤞

**Review by johnstcn — APPROVED** [review]

File: .github/scripts/check.sh [hunk 3]
```diff
@@ -17,14 +23,20 @@ for var in "${required_vars[@]}"; do
     fi
 done
 
+LATEST_REDEPLOY_FAILED="${LATEST_REDEPLOY_FAILED:-0}"
```
Comment (bcpeinhardt) [inline .github/scripts/check.sh:26]: We can remove this

**Review by bcpeinhardt — APPROVED** [review]

**Pull request merged**